### ISSUE TITLE
Is there any REST API

### ISSUE THREAD

**Issue by samouds** [aptabase/aptabase#80]
Hello,

Just discovered aptabase, and as Plausible inspires it, I was to ask if there is any REST API that we can use to build custom charts to add to an existing dashboard.

Thanks

**Comment by goenning** [comment]
Hi @samouds 

there’s currently no API to get the stats, but that’s something I want to add for surr

**Comment by samouds** [comment]
Hello @goenning and thanks for the reply.

It will be really great and can be a good replacement of appcenter.

Do you have any milestone date ?

**Comment by goenning** [comment]
I don't have a date on it yet, but that's a common request, so hopefully soon 🤞

**Comment by nabeelio** [comment]
Yep, just found this and looks like a great replacement for appcenter for my C# desktop apps - at least the analytics portion.
